bigframes/pandas/__init__.py

```diff
@@ -175,6 +175,13 @@ def merge(
     )
 
 
+options = config.options
+"""Global :class:`~bigframes._config.Options` to configure BigQuery DataFrames."""
+
+option_context = config.option_context
```

Comment:
Need to add this to `__all__`: https://github.com/googleapis/python-bigquery-dataframes/blob/ac44ccd3936cdb28755d2bbe16377d489f08d5e5/bigframes/pandas/__init__.py#L472

Would also be good to defined below near the `options` definition.

Reply:
Done